.circleci/config.yml

```diff
@@ -170,6 +170,71 @@ jobs:
       - store_artifacts:
           path: monitor-ci/test-artifacts/results/remocal
           destination: test_artifacts/results/remocal
+
+  oss-changelog:
+    docker:
+      - image: quay.io/influxdb/changelogger:0dbf37ba8a912460358d322dcad414b80ce074f2
+    steps:
+      - checkout
+      - run:
+          name: Generate changelog
+          command: |
+            mkdir release
+            previous_tag="$(git tag -l --merged | egrep '^OSS-[0-9]+\.[0-9]+\.[0-9]+.*' | sort -V | tail -2 | head -1)"
+            GIT_REPO=https://github.com/influxdata/ui git-cliff -t ${CIRCLE_TAG} "${previous_tag}"..${CIRCLE_TAG} > release/CHANGELOG.md
+      - persist_to_workspace:
+          root: .
+          paths:
+            - release/CHANGELOG.md
+      - store_artifacts:
+          path: release/
+
+  build-oss-ui:
+    docker:
+      - image: cimg/node:16.13.0
+    resource_class: large
+    steps:
+      - checkout
+      - run:
+          name: Build UI
+          no_output_timeout: 30m
+          command: |
+            yarn install && yarn build
+            mkdir release
+            tar -czvf release/build.tar.gz ./build
+            cd release && shasum -a 256 build.tar.gz > sha256.txt && cd ../
+      - persist_to_workspace:
+          root: .
+          paths:
+            - release/build.tar.gz
+            - release/sha256.txt
+      - store_artifacts:
+          path: release/
+
+  publish-oss-release:
+    docker:
+      - image: cimg/go:1.17.2
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: Install ghr
+          command: |
+            go install github.com/tcnksm/ghr@v0.14.0
+      - run:
+          name: Push draft release
+          command: |
+            ghr \
+              -t ${TEAM_EDGE_BOT_GITHUB_TOKEN} \
```

Comment:
any concerns about doing this in a public repo? someone could theoretically branch the UI repo and modify this file so that it creates a new draft release. Totally fine if it's not a concern, just checking

Reply:
Hmmm that's a good point. I don't think it would be a concern but I am a bit hesitant now to take the automation this far. I'll have to think about this a bit more - I won't be able to respond fully for a bit, so I'll convert this to a draft for now.